### PR TITLE
Rephrased my Addition

### PR DESCRIPTION
**Pull request by bemijonathan**
> Simple and beginner friendly tutorial for getting started with Vuejs 
> 
> This youtube series covers the entire basics of Vuejs and important topics such as the Vue Instance, Event Listeners, Data binding, Life cycle Hooks and Setting up the vue-router plugin.

### CODE REVIEW

**Comment by dobromir-hristov** [comment]
Dude.. you opened and closed 4 PRs... just push to the same branch, the PRs will get updated....

File: docs/guide/learning/courses.md [hunk 1]
```diff
@@ -119,3 +119,19 @@ The video series starts by building the Vue app from scratch and then moving it
...
+
+## Vue Js 2 Tutorial
+**Author:** The Net Ninja
+
+Simple and beginner friendly tutorial for getting started with Vuejs 
+
+This youtube series covers the entire basics of Vuejs and important topics such as the Vue Instance, Event Listeners, Data binding, Life cycle Hooks and Setting up the vue-router plugin.
+
+<useful-links>
+<useful-links-section title="Official">
```
Comment (dobromir-hristov) [inline docs/guide/learning/courses.md]: This is not an official learning resource.

File: docs/guide/learning/courses.md [hunk 2]
```diff
@@ -119,3 +119,19 @@ The video series starts by building the Vue app from scratch and then moving it
...
+ 
+[YouTube](https://www.youtube.com/playlist?list=PL4cUxeGkcC9gQcYgjhBoeQH7wiAyZNrYa)
+
+<useful-links>
```
Comment (dobromir-hristov) [inline docs/guide/learning/courses.md]: Your closing tags are wrong.

**Comment by bemijonathan** [comment]
opps sorry network providers

File: docs/guide/learning/courses.md (hunk 1 above)
Comment (dobromir-hristov) [reply docs/guide/learning/courses.md]: Ahh others are also stated official fml .... ok leave it for now.

File: docs/guide/learning/courses.md [hunk 3]
```diff
@@ -119,3 +119,20 @@ The video series starts by building the Vue app from scratch and then moving it
...
+
+This youtube series covers the entire basics of Vuejs and important topics such as the Vue Instance, Event Listeners, Data binding, Life cycle Hooks and Setting up the vue-router plugin.
+
+<useful-links>
+<useful-links-section title="Official">
+ 
+[YouTube](https://www.youtube.com/playlist?list=PL4cUxeGkcC9gQcYgjhBoeQH7wiAyZNrYa)
+
+<useful-links-section title="Official">
+<useful-links>
```
Comment (dobromir-hristov) [inline docs/guide/learning/courses.md]:
```suggestion
</useful-links-section>
</useful-links>
```

**Pull request merged**